packages/kanban/src/components/Kanban.svelte

```diff
@@ -261,9 +257,7 @@
       cursor: grab;
     }
     &.dragged {
-      padding: 1rem;
```

Comment:
Why? this was done by purpose, to make all of them look similar during dragging.

Reply:
I think this padding is to compensate the border, which I have also removed from class, the border was the same color as a card-container or unset, so I think we can just remove it at all, and without the border this padding is also not needed.